site/content/en/docs/manual/advanced/ai-tools.md

```diff
@@ -240,34 +240,38 @@ To annotate with trackers, do the following:
 
 To annotate with trackers, do the following:
 
-1. On the menu toolbar, click **OpenCV**![OpenCV](/images/image201.jpg) and wait for the library to load.
+1. Create basic rectangle shapes for tracker initialization
+
+2. On the menu toolbar, click **OpenCV**![OpenCV](/images/image201.jpg) and wait for the library to load.
 
    <br>![](/images/image198.jpg)
 
-2. Go to the **Tracker** tab, select the label, and click **Tracking**.
+3. From **Tracker** drop-down, select tracker and Click **Track**
+
+   <br>![Start tracking an object](/images/tracker_mil_control.png)
+
+4. Annotation actions window will pop-up. Setup `Target frame`
+and `Convert rectangle shapes to tracks` parameters and click `Run`
 
-   <br>![Start tracking an object](/images/image242.jpg)
+   > **Note:** Tracking will be applied to all filtered rectangle annotations.
 
-3. From the **Label** drop-down, select the label for the object.
-4. From **Tracker** drop-down, select tracker.
-5. Click **Track**.
-6. To move to the next frame, on the top menu click the **Next** button (or **F** on the keyboard).
+   <br>![Tracker action](/images/tracker_mil_action.png)
 
-All annotated objects will be automatically tracked when you move to the next frame.
+All annotated objects will be automatically tracked up until target frame parameter.
 
 ### When tracking
 
 - To enable/disable tracking, use **Tracker switcher** on the sidebar.
 
-  ![Tracker switcher](/images/tracker_switcher.jpg)
+  ![Tracker switcher](/images/tracker_switcher.png)
```

Comment:
May you please check if the functionality still working?
It is anyway not applicable for OpenCV tracker, thus it would be prior the previous section

Reply:
Or previous section should be after AI Trackers models

Reply:
Well, it seems the functionality its a bit broken. I was able to stop tracking, but when i click it back, nothing happens. Tracking starts only if I move the track.
![tracking-stop](https://github.com/user-attachments/assets/ab4fd3ba-b03f-439e-b109-0c24267881bc)

Reply:
Moved the section about this button after AI section